Formula/py2gobject3.rb

```diff
@@ -1,5 +1,5 @@
-class Pygobject3 < Formula
-  desc "GNOME Python bindings (based on GObject Introspection)"
+class Py2gobject3 < Formula
```

Comment:
Name feels a little weird considering the upstream project name. What about `Pygobject3` and `Pygobject3-python3` or `Pygobject3-py3`? I think we can leave the "default" one to be Python 2 until `brew install python` installs Python 3.

Reply:
Isn't upstream project name `pygobject`? So I think we want `pygobject` and `pygobject-python3`.

Reply:
@tschoonj those should probably be `pycairo` and `pycairo-python3` if we go by the same convention.

Reply:
There is already a `pygobject` formula for the 2.x.y release series, so I cannot rename it to that.

Reply:
`pygobject@2` and `pygobject` and `pygobject-python3` I think?

Reply:
Using the same logic, you could also do `gtk+ => gtk+@2` and `gtk+3 => gtk+`. 

I could live with renaming `pygobject` to `pygobject2` though.

Reply:
@tschoonj Yes, I'd agree with that renaming too.

Reply:
I am not really a fan of moving to `pygobject` and `pygobject-python3`, as it gives the impression that they are not both on equal footing: to me at least it looks like the latter is a special case of the former. It also raises the question what will be done when python2 support gets dropped: will `pygobject-python3` then again be renamed to `pygobject`?

Reply:
> It also raises the question what will be done when python2 support gets dropped: will pygobject-python3 then again be renamed to pygobject?

Yep. Python 2 is our default Python so this should be applied consistently through our naming. When Python 2 is EOL we'll make Python 3 the default and drop Python 2 support on all packages.

Reply:
Personally I'd vote for the original proposal: `py2gobject3` and `py3gobject3`